From c43081ba56bd7fe4bbaa021251eea8053c5b4e7a Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Wed, 17 Feb 2021 13:52:16 +0200
Subject: [PATCH] [SW-20154] include stdint.h file

In the next patch, the compiler is being upgraded
for Gaudi2 platform with GNU toolchain. For that
toolchain, to avoid compile errors like "unrecognised
types of uint*_t, this patch includes stdint.

Change-Id: I12975bf043ef5adaa182348a4b5c37bd49521a3a
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/common/efuse.h b/common/efuse.h
index 6417f6e..8ce9f7f 100644
--- a/common/efuse.h
+++ b/common/efuse.h
@@ -8,6 +8,8 @@
 #ifndef _EFUSE_H_
 #define _EFUSE_H_
 
+#include <stdint.h>
+
 #define EFUSE_READ_DONE				0x2
 #define EFUSE_NOT_READ				0x1
 
diff --git a/include/arcboot.h b/include/arcboot.h
index 98ad433..7526f67 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -9,6 +9,7 @@
 #define _ARCBOOT_H_
 
 #include <stdio.h>
+#include <stdint.h>
 #include "soc.h"
 
 #define BIT(nr)					(1 << (nr))
